### ISSUE TITLE
v0.15.0 filesystem mounts with different permissions

### ISSUE THREAD

**Issue by jwandrews** [kubernetes/minikube#1003]
**Is this a BUG REPORT or FEATURE REQUEST?**: Bug

**Minikube version**: 0.15.0

**Environment**:
- **OS**: macOS 10.12.2
- **VM Driver**: virtualbox
- **Docker version**: 1.12.6
- **Install tools**: brew

**What happened**: I upgraded from Minikube 0.14.0 to 0.15.0 and now the filesystem mounts with different permissions. It used to be owned by `docker:staff` and now it's `root:vboxsf` and no global read permissions. My pods can't read anything from `hostPath`.

**What you expected to happen**: Expected filesystem to be mounted as `docker:staff` with read permissions to filesystem.

**How to reproduce it** (as minimally and precisely as possible): Upgrade from 0.14 to 0.15

**Comment by aaron-prindle** [comment]
What directory on the host are you referring to?

**Comment by jwandrews** [comment]
@aaron-prindle - `/Users`

**Comment by jpswade** [comment]
:+1: I too was faced with this problem, when I rolled back to v0.14.0, killed the minikube and started it again, everything worked fine.

```
curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.14.0/minikube-darwin-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
```

**Comment by georgecrawford** [comment]
@dlorenc @aaron-prindle - any idea when we might see a v0.16.0 with this fix? Thanks!

**Comment by dlorenc** [comment]
We'll try to get one out this week. Right now you can use the new version of the ISO by passing:

`--iso-url=https://storage.googleapis.com/minikube/iso/minikube-v1.0.4.iso`

to `minikube start`

**Comment by aaron-prindle** [comment]
Closing, fixed with v1.0.6 of the iso which will the default in the release later today.
